hexColor.json

```
@@ -0,0 +1,7 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "./hexCode.json",
+    "title": "Hex Code",
+    "type": "string",
+    "pattern": "^#(?:[0-9a-fA-F]{2}){3,4}$"
```

```suggestion
    "pattern": "^(?:[0-9a-fA-F]{2}){3,4}$"
```
SM doesnt use hash at the start of colors